test/tests/pbs_smoketest.py

```diff
@@ -327,6 +327,7 @@ def test_limits(self):
         self.server.expect(JOB, a, attrop=PTL_AND)
 
     @skipOnCpuSet
+    @skip('This test is broken for oss')
```

Comment:
Don't skip this test.  If you do, it will likely not be unskipped when it is fixed.

Reply:
this is a smoketest, how is it broken?

Reply:
Sorry, this wasn't supposed to be in the PR, just for testing.
It's broken due to #1267